lib/livebook_web/live/session_live.ex

```diff
@@ -1934,6 +1965,16 @@ defmodule LivebookWeb.SessionLive do
     end
   end
 
+  defp after_operation(
+         %{assigns: %{canvas_pid: pid}} = socket,
+         _prev_socket,
+         {:move_output_to_canvas, _client_id, _cell_id}
+       )
+       when not is_nil(pid) do
+    send(pid, {:new_data, socket.private.data.notebook.canvas_settings})
+    socket
+  end
```

Comment:
Let's not tie the LVs together. The canvas should subscribe to data updates and work effectively as a separate client (which is very similar to what AppSessionLive does).

Later we could move the markup to a component, so in the embedded case we don't use separate LV, but we need the separate LV either way, so for now let's do just that.